### PR TITLE
Only pull docker daemon proxy image in tests

### PR DESCRIPTION
**Pull request by shanejbrown**
### Merge requirements satisfied?
- [x] I have updated the documentation or no documentation changes are required.
- [ ] I have added tests to cover my changes.
- [x] I have updated the base version in ``setup.py`` (if appropriate).

### CODE REVIEW

File: tests/test_runner.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+from buildrunner.docker.daemon import DAEMON_IMAGE_NAME
+from buildrunner import docker as buildrunner_docker
```
Comment (shanejbrown) [inline tests/test_runner.py:2]: The ordering of these imports seems strange to me. But Ruff forced me to put these at the top.

**Review by bluesliverx — APPROVED** [review]

File: tests/test_runner.py (hunk 1 above)
Comment (bluesliverx) [reply tests/test_runner.py:2]: Uhh, that's definitely wrong, why would ruff tell you that?

Comment (shanejbrown) [reply tests/test_runner.py]: `# noqa: E402` to the rescue.

File: tests/test_runner.py [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-from buildrunner import docker as buildrunner_docker
 import os
 import sys
 import docker.errors
 
 sys.path.insert(0, os.path.realpath(os.path.join(os.path.dirname(__file__), "..")))
 
 import buildrunner.config.loader  # noqa: E402
+from buildrunner.docker.daemon import DAEMON_IMAGE_NAME  # noqa: E402
+from buildrunner import docker as buildrunner_docker  # noqa: E402
```
Comment (shanejbrown) [inline tests/test_runner.py:9]: Fixed with `# noqa: E402`

**Pull request merged**